### PR TITLE
Rebrand

### PR DESCRIPTION
**Pull request by martincoote**
Cards:

amend email addreses
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12736

Directory intro copy changes https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12943

Update Welcome copy https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12738

Update login copy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12740

Update site header copy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12737

Update membership policy
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12739

Logo swap and favicon addition
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12733

Resource upload copy changes 
https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12987

To test:
This is mainly copy changes with a few assets swapped and added. 

The commits on this branch should map closely to the order of the cards linked above.

Work through these and check all requested changes have been made/ the copy matches that on the cards.

### CODE REVIEW

File: static/favicons/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        {
+            "src": "/android-chrome-384x384.png",
+            "sizes": "384x384",
+            "type": "image/png"
+        }
+    ],
+    "theme_color": "#ffffff",
+    "background_color": "#ffffff",
+    "display": "standalone"
+}
```
Comment (arh30) [inline static/favicons/site.webmanifest]: this looks like a pwa manifest...but a slightly incomplete and and stored in favicons. What's this for?

File: templates/accounts/registration.html [hunk 2]
```diff
@@ -3,26 +3,26 @@
...
+    <p>The Campaigns Library is also a resource in and of itself - our own platform and information architecture system is fully open source. <strong>Your organisation can use the Library as its own resource management tool, and maintain either a public or private collection.</strong> Note that there are <a href="/terms/">Terms and Conditions of Use</a> applicable.</p>
+    <p>If you already belong to an Organisation with a collection please indicate this when you register, and the Librarian will confirm your access to the relevant Organisations.</p>
+    <p>If your organisation would like its own private collection, or to upload resources to the public Library, get in touch with the Librarian at <a href="mailto:librarian@the-open.org">librarian@the-open.org</a></p>
   </div>
 {% endblock main_content %}
```
Comment (arh30) [inline templates/accounts/registration.html]: would prefer to have long paragraphs broken up, but not a biggy !

Comment (martincoote) [reply templates/accounts/registration.html]: yeah i'll fix that i've been inconsistent with this.

File: static/favicons/site.webmanifest (hunk 1 above)
Comment (martincoote) [reply static/favicons/site.webmanifest]: I used https://realfavicongenerator.net/ to generate this icon set. i believe this is used for saving to the home screen on android. so probably the same as a pwa manifest yes. Real favicon generator walks you through a process where you can customise the icon presentation on a few different devices/contexts.

Comment (arh30) [reply static/favicons/site.webmanifest]: ah okay i think this is the slimmed down version, so not a PWA but may set the bookmark icon?

**Comment by martincoote** [comment]
Ready to merge once @alistairclark has finished this card. https://favro.com/organization/b03d6d6d9b11b61167ac4246/54c71b351e54e7aebff9355f?card=Dev-12735

**Pull request merged**